frame/support/src/dispatch.rs

```diff
@@ -306,6 +306,9 @@ impl<T> Parameter for T where T: Codec + EncodeLike + Clone + Eq + fmt::Debug {}
 /// * `integrity_test`: Executes in a test generated by `construct_runtime`, note it doesn't
 ///   execute in an externalities-provided environment. Implement
 ///   [`IntegrityTest`](./trait.IntegrityTest.html) trait.
+/// * `check_inherent_data`: check block in regards to inherent data provided. This is typically
```

Comment:
Why do you want to move this here?

Could you explain what your idea is here for the implementation?